README.md

```diff
@@ -42,21 +42,46 @@ so the request won't be rejected because of a missing/invalid schema. When using
 recommended to provide a `rootValue` to return a fitting value. Examples for these requests can be found in the 
 integration test in the `GraphQLServer.integration.test.ts` class in the `tests` folder. 
 
-## Disable introspection
+## Schema Validation and Disable introspection
+
+Validation rules can be used to define how the `GraphQLServer` should behave when validating the request against the given
+schema. 
+To ease the use `GraphQLServer` uses the `specifiedRules` from `graphql-js` library. If you don't want to use the default
+validation rules you can overwrite them by setting `defaultValidationRules` option to `[]`.
+
+**Warning!**
+Setting both `defaultValidationRules` and `customValidationRules` options to `[]` will disable validation. This might 
+result in unexpected responses that are hard to use for requestors like api users or Frontends.
```

Comment:
Don't you uppercase abbreviations like API?
On the other hand I think you would lowercase the word "frontend"...

Reply:
Fixed